### DISCUSSION TITLE
When I export a text file, it is not visible to whom the person responds.

### DISCUSSION THREAD

**Discussion by pasynnn** [Tyrrrz/DiscordChatExporter#810] [Q&A]
Is it possible to add a function in which, when exporting a text file, it was visible to whom the person was responding. Or is it already there and I just haven't found it?

**Comment by Tyrrrz** [comment]
Text format is quite limited. Use HTML for a more complete feature set.

**Reply by pasynnn** [reply]
Then it is impossible to add this function in any way: with a brief explanation, it is tedious for me to get a chat and get rid of the messages that were answered. Only isolated messages are tedious. Is there any way to solve this problem?

**Reply by Tyrrrz** [reply]
I don't understand what you're asking exactly. But one way or another, plain text format is very limited because you only have text to work with and it's hard to balance it in a way where it doesn't become too noisy and overwhelming. If you need to see replies, you'd have to use HTML.

**Reply by pasynnn** [reply]
I'd like to get text without replies

**Reply by Tyrrrz** [reply]
Currently it's not possible. If a lot of people express interest, it might be added, but for now it sounds like an obscure use case.
